internal/management/controller/slots/slots.go

```diff
@@ -104,7 +104,7 @@ func (sm PostgresManager) List(
 
 	rows, err := db.QueryContext(
 		ctx,
-		`SELECT slot_name, slot_type, active, restart_lsn FROM pg_replication_slots
+		`SELECT slot_name, slot_type, active, coalesce(restart_lsn::text, '') as restart_lsn FROM pg_replication_slots
```

Comment:
That is unnecessary as a null value is an empty string when scanned as a string value.